python/tskit/trees.py

```diff
@@ -32,19 +32,20 @@
 import itertools
 import math
 import warnings
-from dataclasses import dataclass
 from typing import Any
 from typing import NamedTuple
 from typing import Optional
 from typing import Union
 
 import numpy as np
+from dataclasses import dataclass
```

Comment:
This is in the wrong group, should be back where it was

Reply:
For some reason one of the precommit hooks is shuffling this import around on my end (`reorder-python-imports`). I guess I can just disable this next time I push.

Reply:
nah, go with what the pre-commit hooks say!